### PR TITLE
[Improvement-11913] Mask password when creating/editing alert instances

### LINKED ISSUE #11913: [Improvement] [Alert] should mask the password of email password
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

when create alert instance, email password display in plaintext, should mask password
image

### What you expected to happen

mask the email password when create or update alert instance

### How to reproduce

create or update new alert instance

### Anything else

_No response_

### Version

3.0.0

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by c3Vu**
## Purpose of the pull request

- Fixes #11913 

## Brief change log

See above

## Verify this pull request

image

### CODE REVIEW

**Review by EricGao888 — DISMISSED** [review]
LGTM, waiting for CI to pass

**Comment by EricGao888** [comment]
@c3Vu Could u plz pull the latest dev code and rebase this PR on dev? I think this will help CI pass. 
image

**Comment by zhongjiajie** [comment]
this dead link will be fix in https://github.com/apache/dolphinscheduler/pull/14406 but it seem 14406 is also not ready to merge

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-email/src/test/java/org/apache/dolphinscheduler/plugin/alert/email/EmailAlertChannelTest.java [hunk 1]
```diff
@@ -107,7 +106,7 @@ public String getEmailAlertParams() {
                 .setValue("dolphinscheduler@126.com")
                 .build();
 
-        PasswordParam mailPassword = PasswordParam.newBuilder(MailParamsConstants.NAME_MAIL_PASSWD, "passwd")
+        InputParam mailPassword = InputParam.newBuilder(MailParamsConstants.NAME_MAIL_PASSWD, "passwd")
                 .setPlaceholder("if enable use authentication, you need input password")
                 .setValue("escheduler123")
```
Comment (zhongjiajie) [inline dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-email/src/test/java/org/apache/dolphinscheduler/plugin/alert/email/EmailAlertChannelTest.java]: does `escheduler123 ` is a default value? if also we should remove it or change to `dolphinscheduler`

I guess `escheduler` is for `easy scheduler` is pervious name of dolphinscheduler

**Comment by zhongjiajie** [comment]
approval ci run

**Review by zhongjiajie — APPROVED** [review]

**Comment by zhongjiajie** [comment]
Hi @c3Vu , it is a great step for you to become part of dolphionscheduler contributors community 🎉 .
If you want to continue your contribute but could not find issues, maybe you could start in https://github.com/apache/dolphinscheduler/contribute
or just search our issue list https://github.com/apache/dolphinscheduler/issues, Looking for something you interesting.

**Pull request merged**